core/dbt/parser/schemas.py

```diff
@@ -127,6 +128,21 @@ def resource_type(self) -> NodeType:
         return NodeType.Test
 
     def get_paths(self):
+        # TODO: In order to support this, make FilesystemSearcher accept a list
+        # of file patterns. eg: ['.yml', '.yaml']
```

Comment:
You should be able to just `return itertools.chain(yaml_files, yml_files)`. It won't be the most efficient thing in the world, but that's probably fine.